From: Robert Luberda <robert@debian.org>
Date: Tue, 23 Mar 2010 22:04:47 +0100
Subject: 03-manpages

Added README.Debian file and the 'Debian note' to sa1 and sa2 man pages
to explain source of common problems (closes: #279049, #269712).
---
 man/sa1.in | 6 +++++-
 man/sa2.in | 5 +++++
 2 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/man/sa1.in b/man/sa1.in
index 7f6f88d..20f6730 100644
--- a/man/sa1.in
+++ b/man/sa1.in
@@ -63,7 +63,11 @@ To collect data (including those from disks) every 10 minutes,
 place the following entry in your root crontab file:
 
 .B 0,10,20,30,40,50 * * * * @SA_LIB_DIR@/sa1 1 1 -S DISK
-
+.SS Debian note
+The Debian sysstat package has already placed such an entry in your system crontab.
+Please refer to the
+.I /usr/share/doc/sysstat/README.Debian
+file for details.
 .SH FILES
 .I @SA_DIR@/saDD
 .br
diff --git a/man/sa2.in b/man/sa2.in
index b27f102..ddd7931 100644
--- a/man/sa2.in
+++ b/man/sa2.in
@@ -50,6 +50,11 @@ in the
 .I @SA_DIR@
 directory, where the DD parameter is a number representing the day of the
 month.
+.SS Debian note
+The Debian sysstat package has already placed such an entry in your system crontab.
+Please refer to the
+.I /usr/share/doc/sysstat/README.Debian
+file for details.
 .SH FILES
 .I @SA_DIR@/sarDD
 .br
